sourcetools/com.ibm.uma/com/ibm/j9/uma/configuration/ConfigurationImpl.java

```diff
@@ -43,6 +43,7 @@
 import com.ibm.j9.uma.platform.PlatformUnix;
 import com.ibm.j9.uma.platform.PlatformWindows;
 import com.ibm.j9.uma.platform.PlatformZOS;
+import com.ibm.j9.uma.platform.PlatformOSX;
```

Comment:
Can you please organize imports (alphabetically)?

Reply:
Done, in the next push.